test/Carter.Tests/Modelbinding/TestModel.cs

```diff
@@ -15,5 +16,7 @@ public class TestModel
         public IEnumerable<int> MyIntArrayProperty { get; set; }
 
         public IEnumerable<int> MyIntListProperty { get; set; }
+
+        public Guid MyGuidListProperty { get; set; }
```

Comment:
Can you rename this to `MyGuidProperty` please?

Reply:
Yeap! Will do.

Reply:
Done.